subxt/src/rpc/rpc_client_t.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+use crate::error::RpcError;
+use futures::Stream;
+use serde_json::value::RawValue;
+use std::{
+    future::Future,
+    pin::Pin,
+};
+
+/// Any RPC client which implements this can be used in our [`super::Rpc`] type
+/// to talk to a node.
+///
+/// This is a low level interface which expects an already-serialized set of params,
+/// and returns an owned but still-serialized [`RawValue`], deferring deserialization to
```

Comment:
RawValue is quite hard to deal with in general and it would be good if you could point to `RpcParams` for an example implementation of this.

My noob eyes find it hard to look at this trait along with the docs of `RawValue` to understand how to do it

Reply:
That makes sense; I'll add some more examples to show how these methods can be called manually!

Reply:
(I just ended up pointing to RpcParams in the docs to show examples of building params)